x-pack/qa/third-party/active-directory/build.gradle

```diff
@@ -21,27 +18,22 @@ forbiddenPatterns {
 }
 
 test {
+  Project fixtureProject = project(":x-pack:test:smb-fixture")
   /*
    * We have to disable setting the number of available processors as tests in the same JVM randomize processors and will step on each
    * other if we allow them to set the number of available processors as it's set-once in Netty.
    */
+  dependsOn fixtureProject.composeUp
   systemProperty 'es.set.netty.runtime.available.processors', 'false'
   include '**/*IT.class'
   include '**/*Tests.class'
+  doFirst {
+    fixtureProject.dockerCompose.servicesInfos.fixture.ports.forEach { container, host ->
+      logger.info(" adding system property fixture.smb-fixture.ports.$container -> $host")
```

Comment:
nit: can you use `${container}` so it is clear where the substitution ends vs string value? Same below with the actual system property.